akash1/HTML/ASSIn3/contact.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>contact</title>
+    <link rel="stylesheet" href="Assignment3Q2.html">
+    <link rel="stylesheet" href="MI9.html">
+
+<body>
+
+    <style>
+        body {
+            background-color: seagreen;
+            background-repeat: no-repeat;
+            background-size: cover;
+            background-attachment: fixed;
+
+
+        }
+    </style>
+    </head>
+
+    <body>
+        <center>
```

Comment:
don't use center as a tag. it is older version

Reply:
mam , i  am stuck for this changes. i am using this assignment only internal css and not using any bootstrap template so help me which tag is to use this replace center.. plz help me

Reply:
u can use style text-align: center.